website/src/blogs/a-primer-on-openebs-continuous-integration.md

```diff
@@ -32,7 +33,7 @@ The e2e phase involves running several parallel pipelines (based on Kubernetes c
 
 ![OpenEBS CI Workflow](/images/blog/openebs-ci-flow.jpg)
 
-**Note**: *Currently, the GitLab CI works in a “retrospective mode,” as it is invoked on commits to the upstream branches. There is work in progress to extend the support for pull requests (presently, a travis-based build verifies commit sanity to aid PR acceptance).*
+**Note**: _Currently, the GitLab CI works in a “retrospective mode,” as it is invoked on commits to the upstream branches. There is work in progress to extend the support for pull requests (presently, a travis-based build verifies commit sanity to aid PR acceptance)._
```

Comment:
Why this change? Using asterisks is the recommended method for text styling https://www.markdownguide.org/basic-syntax/